web/client/plugins/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+/**
+* @overview list of plugins used for mapstore2
+* @name plugins
+* @module plugins
```

Comment:
This do not include anything. It's confusing. We should include the available plugins.

The reason why the plugins included in the product is to optimize the bundle with only the plugins included. 
Here we can link the available plugins with links to the documentation of each of them.